### ISSUE TITLE
bugsnag.ErrorClass is ignored on v1.4.0

### ISSUE THREAD

**Issue by terashi58** [bugsnag/bugsnag-go#98]
### Expected behavior
Reported error class is specified by `bagsnag.ErrorClass` in metadata.

### Observed behavior
Reported error class is the reflected type name of `error`.

### Steps to reproduce
```.go
err := errors.New("msg")
bugsnag.Notify(err, bugsnag.ErrorClass("ExpectedErrorClass"))
```

### Version
v1.4.0

### Additional information
It worked until v1.3.2.
I believe [the order of concatenated raw data in Notifier](https://github.com/bugsnag/bugsnag-go/blob/v1.4.0/notifier.go#L71) and [the Event construction by error](https://github.com/bugsnag/bugsnag-go/blob/v1.4.0/event.go#L133) caused this issue.

**Comment by ibrt** [comment]
Ran into this issue as well, and diagnosed it in the same way. Please fix, it's very annoying.

**Comment by kinbiko** [comment]
Hi @terashi58, @ibrt,

Thanks for flagging this one up. Apologies for this, I can see how this is an annoying bug and I'll try and get this fixed in the next couple of weeks. In the meantime, the workaround @ibrt suggested should do the trick.

**Comment by kinbiko** [comment]
Hi again,

Sorry about the wait. This issue is now fixed in v1.5.1.
